app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -244,25 +264,127 @@ private fun RecentSearchGroupItem(
     }
 }
 
+/**
+ * Displays a recent synced tab.
+ *
+ * @param tab Optional synced tab. If null, displays placeholders.
+ * @param onRecentSyncedTabClick Invoked when item is clicked.
+ * @param onSeeAllClick Invoked when "See all" button is clicked.
+ */
+@Suppress("LongMethod")
+@Composable
+private fun RecentSyncedTabItem(
+    tab: RecentTab.SyncedTab?,
+    onRecentSyncedTabClick: (RecentTab.SyncedTab) -> Unit,
+    onSeeAllClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .requiredHeight(180.dp)
+            .clickable { tab?.let { onRecentSyncedTabClick(tab) } },
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layer2,
+        elevation = 6.dp
+    ) {
+        Column(modifier = Modifier.padding(16.dp), verticalArrangement = Arrangement.SpaceBetween) {
+            Row(modifier = Modifier.height(IntrinsicSize.Min)) {
+                if (tab == null) {
+                    RecentTabImagePlaceholder()
+                } else {
+                    RecentTabImage(
+                        tab = tab,
+                        modifier = Modifier
+                            .size(108.dp, 80.dp)
+                            .clip(RoundedCornerShape(8.dp))
+                    )
+                }
+
+                Spacer(modifier = Modifier.width(16.dp))
+
+                Column(
+                    verticalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier.fillMaxHeight()
+                ) {
+                    if (tab == null) {
+                        RecentTabTitlePlaceholder()
+                    } else {
+                        RecentTabTitle(title = tab.title)
+                    }
+
+                    Row {
+                        Image(
+                            painter = painterResource(R.drawable.ic_synced_tabs),
+                            contentDescription = stringResource(
+                                R.string.recent_tabs_synced_device_icon_content_description
+                            ),
+                            modifier = Modifier
+                                .size(18.dp, 18.dp)
```

Comment:
```suggestion
                            modifier = Modifier.size(18.dp)
```
Can just use the single param version of `size()`